dask/dot.py

```diff
@@ -139,8 +140,12 @@ def to_graphviz(
 ):
     if data_attributes is None:
         data_attributes = {}
+    else:
+        data_attributes = copy.copy(data_attributes)
```

Comment:
It seems a little simpler to use `data_attributes.copy()` instead. Is there a reason to use `copy.copy` instead?

Reply:
There are two problems:
1) The arguments are being overwritten on the outer scope by the function. This does not effect the output of the function though.
2) In the function there are problems accessing and overwriting the arguments. This effects the output of the function. The problem is described here: https://github.com/dask/dask/pull/5996#issuecomment-596949529

I want use the fix in version 1b7eb8e, because it fixes both issues. Is that okay? I can use `.copy()`, which is indeed nicer.

Greetings

Reply:
Ah, I missed that commit. Yes, you are right that change will fix your label problem. Since that commit already introduces copy as an import, I am fine with using `copy.copy` here. I think this is ready as soon as tests pass.

Reply:
Why do you recommend this diff? The last three changes (after line 163) solve both problems.

Reply:
You are right. Just the last three are needed.